TESTPLAN.md

```diff
@@ -0,0 +1,77 @@
+ - [ ]  Add Autofill support ([#3241](https://github.com/brave/browser-laptop/issues/3241))
+ - [ ]  Form fill (Autofill) enhancement ([#860](https://github.com/brave/browser-laptop/issues/860))
+ - [ ]  Autofill should not happen on URL bar unless item in autocomplete list is clicked ([#3012](https://github.com/brave/browser-laptop/issues/3012))([#3225](https://github.com/brave/browser-laptop/issues/3225))
+ - [ ]  Editing Autofill entries are not saved. Creates a blank entry([#3368](https://github.com/brave/browser-laptop/issues/3368))
+
+ ### Bookmarks
+ - [ ]  Bookmark title is not updated unbookmarking/rebookmarking bug verified ([#3284](https://github.com/brave/browser-laptop/issues/3284))
+ - [ ]  Adding bookmark will now properly update title + rendering fix ([#3339](https://github.com/brave/browser-laptop/issues/3339))
+ - [ ]  bookmark title (onhover) on bookmark bar cannot be updated ([#3080](https://github.com/brave/browser-laptop/issues/3080))
+ - [ ]  On clean install, adding a bookmark via the tabs tray does not open the bookmark tray. ([#3331](https://github.com/brave/browser-laptop/issues/3331))
+ - [ ]  Fix bookmark toolbar not showing on first bookmark from context menu ([#3355](https://github.com/brave/browser-laptop/issues/3355))
+ - [ ]  Bookmarked page cannot be removed from context menu ([#3332](https://github.com/brave/browser-laptop/issues/3332))
+ - [ ]  If a site does not have a title, the bookmark should default to the url instead. ([#3344](https://github.com/brave/browser-laptop/issues/3344))
+ - [ ]  Bookmarks Menu should list the bookmarks like other browsers do ([#1993](https://github.com/brave/browser-laptop/issues/1993))
+ - [ ]  "auto open bookmarks toolbar for the first bookmark" test needs to be updated ([#2941](https://github.com/brave/browser-laptop/issues/2941))
+ - [ ]  Bookmarks now show under Bookmarks menu ([#3055](https://github.com/brave/browser-laptop/issues/3055))
+ - [ ]  Users can create bookmark folders with no name ([#3188](https://github.com/brave/browser-laptop/issues/3188))([#3190](https://github.com/brave/browser-laptop/issues/3190))
+
+ ### History  
+ - [ ]  about:history contains bookmark items as browsing history which can't be cleared ([#3330](https://github.com/brave/browser-laptop/issues/3330))
+ - [ ]  Bookmarked history entries now show context menu in about:history ([#3336](https://github.com/brave/browser-laptop/issues/3336))
+ - [ ]  Ctrl + Y is not opening history ([#3312](https://github.com/brave/browser-laptop/issues/3312))
+ - [ ]  Updates to history page: ([#3320](https://github.com/brave/browser-laptop/issues/3320))
+ - [ ]  History follow up ([#3253](https://github.com/brave/browser-laptop/issues/3253))
+ - [ ]  Back button history does not hint at options beyond a certain point.  ([#2889](https://github.com/brave/browser-laptop/issues/2889))
+ - [ ]  Implement history and Ctrl +Y to bring it up ([#444](https://github.com/brave/browser-laptop/issues/444))
```

Comment:
Next time you're in this file just adjust the shortcut spacing mentioned here, I think you missed this comment from before.